projects/packages/search/src/instant-search/components/overlay.scss

```diff
@@ -44,7 +44,34 @@ $overlay-horizontal-padding-lg: 3em;
 		padding: $overlay-vertical-padding-lg $overlay-horizontal-padding-lg;
 	}
 
-	h1, h2, h3, h4, h5, h6 {
+	// TODO: look into a more sustainable approach to reduce the number of !important being used to override the button styles
+	button {
+		background-color: inherit !important;
+		color: $color-text-lighter !important;
+		font-size: inherit;
+		font-weight: inherit;
+		padding: inherit;
+		outline-offset: inherit;
+
+		&:focus {
+			outline: 1px auto Highlight;
+			outline: 1px auto -webkit-focus-ring-color !important;
+		}
+	}
+
+	input[type='checkbox'] {
```

Comment:
My original feedback still stands here I think - let's try and target specific classnames where we can, especially now we're using sprinkles of `!important`.

I think we can successfully target the visible buttons with classes (I think only sort + load more are visible) and all the checkboxes have the same class name.